ethereum/contracts/zksync/facets/Base.sol

```diff
@@ -22,7 +22,15 @@ contract Base is ReentrancyGuard, AllowListed {
 
     /// @notice Checks that the message sender is an active governor or admin
     modifier onlyGovernorOrAdmin() {
-        require(msg.sender == s.governor || msg.sender == s.admin, "Only by governor or admin");
+        if (s.governor.code.length > 0) {
```

Comment:
Do we really need this complex structure instead of having two roles in the same contract?

Reply:
What if `s.governor` is a smart contract that doesn't implement Ownable?